core/setup.cfg

```diff
@@ -68,9 +68,9 @@ docs =
     sphinxcontrib-spelling
     pyenchant
     sphinx-autobuild
-    sphinx_rtd_theme
     sphinx-csv-filter
     sphinx_tabs
+    furo
```

Comment:
I have a few CSS additions from the Chaquopy Furo configuration which might be worth including here. But first, can we extract the docs theme change into a separate PR? That should make both PRs easier to follow.

Reply:
Yeah - this PR has become a bit of a runaway encompassing too many things; I need to give some thought to how I can land what is here (either in pieces, or getting what is here to a mergeable state).